Doc/library/ctypes.rst

```diff
@@ -308,12 +308,12 @@ bytes objects are immutable)::
 
    >>> s = "Hello, World"
    >>> c_s = c_wchar_p(s)
-   >>> print(c_s)
-   c_wchar_p('Hello, World')
+   >>> print(c_s, c_s.value)
```

Comment:
Fix this problem, and add a comment to point out that memory location have changed.